requirements/dev.txt

```diff
@@ -1,2 +1,2 @@
-https://github.com/celery/py-amqp/zipball/master
-https://github.com/celery/vine/zipball/master
+https://github.com/celery/py-amqp/zipball/v2.6
```

Comment:
is it really need to be changed?

Reply:
master for those projects has dropped support for Python 2.7 and Python 3.5. This pins the dev-dependencies for the 4.x branch here to the branches in those projects so that kombu 4 (and thus Celery 4) can maintain support for python 2.7 and 3.5 in the spirit of the LTS branch declared with the release of Celery 5.

Reply:
Related, it seems that brotli on windows needs some kind of pinning to make it continue to work for python 2.7 but I didn't want to dig too hard into what that was. That's why appveyor tests are failing.

Reply:
Seems like it will work with 1.0.7 based on this: https://github.com/google/brotli/issues/848